source/docs/articles/sites/code/ldap-and-ldaps.md

```diff
@@ -10,9 +10,9 @@ keywords: ldap, ldaps, ldap and ldaps, using ldap as a provider, ldap provider,
 
 ## LDAP as a Consumer  
 
-Using LDAP as a consumer of services is supported on the platform and will work at all plan levels, assuming correct configuration. The implementation and configuration details are up to the user as not all instances are supported.
+Using LDAP as a consumer of services is supported on the platform and will work at all plan levels, assuming correct configuration. The implementation and configuration details are up to the user as not all instances are supported. For general information about implementing LDAPS, see [https://drupal.org/node/1404368](https://drupal.org/node/1404368) and [https://drupal.org/node/1302032](https://drupal.org/node/1302032).
 
-LDAP provides access and maintenance of a distributed directory storing organized sets of records. LDAP is often used for single sign-on authentication, and many sites on Pantheon use LDAP and LDAPS for authentication. Authentication requests will originate from Pantheon to the LDAP server used (outbound from Pantheon).  
+LDAP provides access and maintenance of a distributed directory storing organized sets of records. LDAP is often used for single sign-on authentication and authentication requests will originate from Pantheon to the LDAP server used (outbound from Pantheon). However, we do not recommend using LDAP for single sign-on authentication as it is not secure.  The recommended approach is SAML (both for sites and the Dashboard) and OneLogin. Learn more about [two-factor authentication](/docs/guides/two-factor-authentication/).

```

Comment:
@nataliejeremy I may be misunderstanding the [original comments](https://github.com/pantheon-systems/documentation/pull/302#issuecomment-85672363), but I don't think LDAP for SSO can be used at all unless it's in conjunction with a SaaS provider (such as onelogin). If so, we should update the paragraph to suggest they use SAML with a SaaS provider to bridge authentication over to LDAP. (Current verbiage doesn't mention bridging back to LDAP and mentions OneLogin directly). 

If a user is on this doc hoping to configure some type of SSO system with LDAP, I think a warning alert is fine. The verbiage used within the warning should be enough to inform the reader it doesn't apply to all implementations of LDAP, but just SSO. 

I defer to you and @bmackinney on that though